portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio | Home Page</title>
+    <link rel="stylesheet" href="style.css"></link>
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="content">
+      <div class="header">
+        <a href="index.html">home page</a>
+        <a href="photogallery.html">my photo gallery</a>
+        <a href="recipes.html">recipes i like</a>
+        <a href="https://github.com/jennygaoo">my github</a>
+        <a href="penguin.html">penguin livestream</a>
+        <a href="aboutme.html">about me</a>
+      </div>
+      <h1>Welcome to Jenny's Portfolio!</h1>
+      <div>
+        <img src="/images/gallery/kermit.gif" alt="kermit dancing"/>
+      </div>
+      <div>
+        <audio controls autoplay>
+          <source src="/audio/rholiday.mp3" type="audio/mpeg" 
+              alt ="nicki minaj roman holiday"> 
+        </audio>
+        <p>
+          the audio has issues autoplaying :,( I think it should work if you
```

Comment:
nit: add a TODO to fix!

Reply:
I wasn't able to find how to do this, and i also came across these specifications about autoplaying
![image](https://user-images.githubusercontent.com/55329941/83359097-ed19c900-a32c-11ea-8689-1d0e1828b62c.png)

Reply:
That's fine. Just add a comment saying `<!-- TODO(jennygaoo): fix audio issues -->` You can go back and fix whenever (even after your internship). It's just a reminder for you to do it when you look at the code.

Also please, please, please don't work on weekends as an intern. :)